README.md

```diff
@@ -624,12 +617,6 @@ fastbootConfigTree() {
 
 The above configuration will be available in Node via the `FastBoot.config()` function. Therefore, in order to get the above config, the addon/app can call `FastBoot.config('<engine-name>')`.
 
-## Known Limitations
```

Comment:
@steveclifton Maybe my personal preference, but I would keep this header as otherwise below section will be part of "Build Hooks for FastBoot" which not exactly what it is.
What I mean, we can keep the header "## Known Limitations" and only get rid of the statement below it

Reply:
I would say that we can just remove the sentence `Only the most brave should even
consider deploying this to production.` and that works. And I agree with @SergeAstapov that we should keep the section